content/blog/documentation-driven-development/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 {
 title: "A Better Way To Code: Documentation Driven Development",
-description: "",
+description: "Test driven development is often taught to improve a your dev workflow; I think works Documentation drive development (DDD) works better.",
```

Comment:
```suggestion
description: "Test Driven Development is often taught to improve a your workflow; I present Documentation Driven Development as an alternative approach.",
```
Noticed a small typo here - suggesting a slight rephrasing of this as well as fixing that.